control-plane/mbus-api/src/message_bus/v0.rs

```diff
@@ -12,14 +12,14 @@ pub type BusError = ReplyError;
 pub type BusResult<T> = Result<T, BusError>;
 
 macro_rules! only_one {
-    ($list:ident, $resource:expr, $filter:expr) => {
+    ($list:ident, $resource:expr) => {
         if let Some(obj) = $list.first() {
             if $list.len() > 1 {
                 Err(ReplyError {
                     kind: ReplyErrorKind::FailedPrecondition,
                     resource: $resource,
                     source: "".to_string(),
-                    extra: $filter.to_string(),
+                    extra: "".to_string(),
```

Comment:
wouldn't it still be useful to have the actual filter used in the error? Eg, we know if was about a resource, but we don't know which particular one?

Reply:
The REST calls that return these errors were already called with an ID so they are already aware of the resource that they were requesting.